src/SchedulerManager/Travely.SchedulerManager.API/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim AS build
+WORKDIR /src
+COPY ["src/SchedulerManager/Travely.SchedulerManager.API/Travely.SchedulerManager.API.csproj", "src/SchedulerManager/Travely.SchedulerManager.API/"]
```

Comment:
Please rename Travely.SchedulerManager.API to API.
You can see the recommended naming conventions in tour-manager branch.